src/components/WelcomeModal/WelcomeModal.tsx

```diff
@@ -0,0 +1,200 @@
+    subtitle: "Discover your new and improved patches workflow!",
+    description:
+      "We’ve made your patches workflow better by adding more filtering options, reducing load times, and improving the design.",
+  },
+  {
+    img: "patch_gif_06_10.gif",
+    subtitle: "We’ve also updated the patch page!",
+    description:
+      "We’ve made it easier to navigate through your tasks and find the information you’re looking for.",
+  },
+  {
+    img: "newui_gif_06_10.gif",
+    subtitle:
+      "We really hope you enjoy the new UI, but just in case you miss the old Evergreen…",
+    description:
+      "We’re still working every day to make this better, adding new features and new pages all the time. In case you want to opt out of the new UI and miss all the updates... navigate to your preferences to do so.",
+  },
+];
+
+const WelcomeModal = () => {
+  const [visible, setVisible] = useState(true);
+  const [activeSlide, setActiveSlide] = useState(0);
+  const slider = useRef() as React.MutableRefObject<Carousel>;
+  const [updateUserSettings] = useMutation<
+    UpdateUserSettingsMutation,
+    UpdateUserSettingsMutationVariables
+  >(UPDATE_USER_SETTINGS, {
+    onCompleted: () => {
+      setVisible(false);
+    },
+    onError: (err) => {
+      reportError(err).warning();
+    },
+  });
+  const handleWelcomeClosed = async () => {
+    try {
+      await updateUserSettings({
+        variables: {
+          userSettings: {
+            useSpruceOptions: {
+              hasUsedSpruceBefore: true,
+            },
+          },
+        },
+        refetchQueries: ["GetUserSettings"],
+      });
+    } catch (e) {}
+  };
+  return (
+    <Modal
+      footer={
+        <Button
+          variant={Variant.Primary}
+          onClick={handleWelcomeClosed}
+          data-cy="close-welcome-modal"
+        >
+          Done
+        </Button>
+      }
+      onCancel={handleWelcomeClosed}
+      style={{ top: 20 }}
+      visible={visible}
+      width={800}
+      maskStyle={{
+        backgroundColor: black,
+        opacity: 0.9,
+      }}
+      wrapProps={{
+        "data-cy": "welcome-modal",
+      }}
+    >
+      <CardTitle>Welcome to the New Evergreen UI!</CardTitle>
+
+      <Carousel
+        afterChange={(number) => setActiveSlide(number)}
+        autoplay
+        dots={false}
+        slickGoTo={activeSlide}
+        ref={slider}
+      >
+        {carouselCards.map((card) => (
+          <CarouselCard key={`card_${card.img}`} {...card} />
+        ))}
+      </Carousel>
+      <CarouselDots
+        activeSlide={activeSlide}
+        cards={carouselCards}
+        slider={slider}
+      />
+    </Modal>
+  );
+};
+
+type CarouselCard = {
+  img: string;
+  subtitle: string;
+  description: string;
+};
+
+const CarouselCard: React.FC<CarouselCard> = ({
+  img,
+  subtitle,
+  description,
+}) => (
+  <CardContainer>
+    <CardWrapper>
+      <ImgContainer src={`/static/img/welcome_modal/${img}`} />
```

Comment:
I think doing so will potentially break caching in the CDN. I think using the relative path here is correct, since we'll be maintaining the assets ourselves in the repository which should get deployed together

Reply:
Hey @john-m-liu  could we upload the gifs to the CDN directly and just have a link to them in the repo. Including these large gifs as part of the deploy process has a large impact on deploy times

Reply:
It should be possible, though we might find that maintaining them outside the repo is more annoying than it's worth. Search for "Kernel AWS Access" on the internal wiki to find directions on how to access the AWS console, and you can upload them to Spruce's S3 buckets